lib/services/devices/deviceRegistryMongoDB.js

```diff
@@ -30,7 +30,7 @@ const errors = require('../../errors');
 const constants = require('../../constants');
 const Device = require('../../model/Device');
 const async = require('async');
-const context = {
+let context = {
```

Comment:
This looks very strange - it is necessary from #901  - it looks like that PR is amending this constant - is this correct?

Reply:
Not sure... maybe @AlvaroVega could provide feedback on this.

Reply:
I guess there is no problem no move from `const` to `let`

Reply:
So I understand it's correct and NTC